en/Getting started/Sync your notes across devices.md

```diff
@@ -0,0 +1,111 @@
+
+For example, you can use any of the following services:
+
+- Dropbox
+- Google Drive
+- iCloud Drive
+- OneDrive
+- Syncthing
+
+To sync your notes, follow the instructions for the service you're using to sync a folder on your local file system. Then open the folder as an existing vault on all your desktop devices.
+
+## Sync your notes on iPhone and iPad
+
+To sync your notes to your iPhone or iPad, we officially support the following options:
+
+- [[Introduction to Obsidian Sync|Obsidian Sync]]
+- [[#iCloud Drive]]
+
+**Note:** The following services are not supported. If you discover a way to sync your notes on your iOS device using any of these services, let us know on our community channels.
+
+- Dropbox
+- Google Drive
+- OneDrive
+- Syncthing
+
+### iCloud Drive
+
+Obsidian can use iCloud Drive as a local file system.
+
+#### Create a new vault in iCloud Drive
+
+To create a new vault in iCloud Drive on your iPhone or iPad:
+
+1. Tap **Create new vault**.
+2. In **Vault name**, enter the name of your vault.
+3. Enable **Store in iCloud**.
+4. Tap **Create**.
+
+Obsidian has created a new folder inside your iCloud Drive. To open the iCloud Drive folder as a vault on your computer:
+
+1. Open **Obsidian** on your computer.
+2. In the the vault switcher, to the right of **Open folder as vault**, click **Open**.
+3. Navigate to **iCloud Drive** -> **Obsidian**.
+4. Select the folder with the name of the vault you want to sync with.
+
+#### Sync an existing vault with iCloud Drive
+
+To sync an existing vault using iCloud, you need to create an empty vault in iCloud Drive and then move your notes on your other device to the empty vault.
+
+To create a new empty vault in iCloud Drive:
+
+1. Tap **Create new vault**.
+2. In **Vault name**, enter the same name as the vault you want to sync with.
+3. Enable **Store in iCloud**.
+4. Tap **Create**.
+
+To move your notes to the new vault in your iCloud Drive:
+
+1. On your computer, open the **iCloud Drive** folder.
+2. Open the **Obsidian** folder. It might take a few minutes to appear.
+3. Move the files in your existing vault to the folder with the name of your vault.
+
+iCloud syncs the files to your mobile device. Depending on the size of your vault, this might take a few minutes.
+
+### Working Copy
+
+If you store your notes in a [Git](https://git-scm.com/) repository, you may want to look at [Working Copy](https://apps.apple.com/us/app/working-copy-git-client/id896694807)—a Git client for your iOS.
+
+To sync using Working Copy:
+
+1. Create an empty vault on your iPhone or iPad.
+2. Use the **Setup Folder Sync** action and select your empty vault.
+3. Commit and push any changes to your vault using the Working Copy app.
+
+**Note:** While we don't officially support this method, several of our users have reported that they've successfully used Working Copy to sync their notes.
+
+### Why can't I sync using X?
```

Comment:
I think another thing we can mention is that, it is possible to add integration for any third party sync/backup that provides a decent web API, something that most cloud providers actually do have.

It is definitely doable to implement it as an Obsidian community plugin, since Obsidian Sync is implemented as a plugin itself, however given the amount of work required, I haven't seen anyone bothered enough to implement it.